src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useState } from 'react';
```

Comment:
Found one bug when searching for repository recommendation. 

When searching for an EPEL package without adding the EPEL repo there are results listed in Available - Other repos. Adding any of those packages should add the EPEL repo as well which is explained in a modal that will pop up as soon as one of the packages is selected.

Previously the modal popped up only for the first package, but now it pops up twice in a row even though the repo being added should be the same EPEL repo.

![two-modals](https://github.com/osbuild/image-builder-frontend/assets/49452678/b0eb4e94-f897-4bb2-a64e-52145961cd4a)

Reply:
hmmm will look at this.

Reply:
Found the problem, the component wasn't getting rerendered, so wasn't receiving updated props in it's parent function. 

But I also noticed that we should likely only be adding that initial selection if the user "accepts" the modals proposition. 
So added that change.